### ISSUE TITLE
Revamp navigation bar

### ISSUE THREAD

**Issue by Chalarangelo** [30-seconds/30-seconds-web#64]
Depends on #57.

After the blog introduces changes to the navigation bar, we should take the time to redesign it in order to provide a cleaner UI/UX for users. Namely, we might opt for the following:

- Move desktop navigation bar to the top of the screen to better utilize screen real estate.
- Allow swipe gestures if possible to navigate between "screens" (Snippets, Blog)
- Declutter: Remove settings and move them to a dedicated location (currently just Dark Mode, but will probably add default order options (see #23) in there).
- Another idea is to move the navigation bar to the top for all users, as there will only be two options that are swipable. This will also allow us to remove icons entirely.
- Figure out search placement after the revamp (although all of the above suggestions can survive having the search as part of the menu - thus allowing for a 3-button navigation bar instead of a 2-button one).

**Comment by Chalarangelo** [comment]
Closed as a duplicate of #57
